### ISSUE TITLE
Is removing a property from a datacontract Class (not currently used by The client) a breaking Change?

### ISSUE THREAD

**Issue by sabbadino** [protobuf-net/protobuf-net.Grpc#310]
Is removing a property from a datacontract Class (not currently used by The client) a breaking Change for the client?

**Comment by sabbadino** [comment]
I did a repro and it looks like not being a breaking Change (of course i left the Order of the other properties un touched

**Comment by mgravell** [comment]
No, it isn't. You may want to make use of [ProtoReserved] (IIRC, not at PC) to ensure you don't reuse it in unfortunate ways
